RELEASE_NOTES.md

```diff
@@ -1,3 +1,86 @@
+
+##### Aggregating Notifications with a log resource
+
+```ruby
+template '/etc/foo' do
+  source 'bar.erb'
+  notifies :write, 'log[Aggregate notifications using a single log resource]', :immediately
+end
+
+log 'Aggregate notifications using a single log resource' do
+  notifies :restart, 'service[foo]', :delayed
+end
+```
+
+##### Aggregating Notifications with a notify_group resource
+
+```ruby
+template '/etc/foo' do
+  source 'bar.erb'
+  notifies :run, 'notify_group[Aggregate notifications using a single notify_group resource]', :immediately
+end
+
+notify_group 'Aggregate notifications using a single notify_group resource' do
+  notifies :restart, 'service[foo]', :delayed
+end
+```
+
+#### ChefDeprecations/ResourceWithoutNameOrProvides
+
+The `ChefDeprecations/ResourceWithoutNameOrProvides` cop checks for legacy HWRP-style resource that do not set either `resource_name` or `provides`, which are required by Chef Infra Client 16 and later.
+
+#### ChefCorrectness/ChefApplicationFatal
+
+The `ChefCorrectness/ChefApplicationFatal` cop detects cookbooks that use `Chef::Application.fatal!` to raise a failure during a Chef Infra Client run. When an error needs to be presented to a user `raise` should be used instead as it provides the full stack trace to make debugging significantly easier.
+
+#### ChefCorrectness/PowershellScriptDeleteFile
+
+The `ChefCorrectness/PowershellScriptDeleteFile` cop detect using a `powershell_script` resource to delete a file. This should be accomplished by using the `file` resource with the `:delete` action instead. The `file` resource offers a simpler syntax and full idempotency.
+
+#### ChefModernize/UseMultipackageInstalls
+
+The `ChefModernize/UseMultipackageInstalls` cop detects cookbooks that use Ruby loops to install multiple packages using multiple package resources. When using a package provider that supports multi-package installs you can pass an array of packages to install to a single package resource. Passing multiple packages to a single package resource greatly simplifies log output and is significantly faster. Multi-package installs are available for package installs using apt, dnf, yum, snap, dpkg, deb, and Chocolatey package management systems.
+
+#### ChefModernize/ProvidesFromInitialize
+
+The `ChefModernize/ProvidesFromInitialize` cop detects legacy HWRP-style resources that set the provides name in an `intialize` method instead of using the `provides` method in the resource DSL.
```

Comment:
```suggestion
The `ChefModernize/ProvidesFromInitialize` cop detects legacy HWRP-style resources that set the `provides` name in an `intialize` method instead of using the `provides` method in the resource DSL.
```